package.json

```diff
@@ -10,6 +10,7 @@
     "lint": "tslint -p tsconfig.json",
     "prepare": "npm run build",
     "prepublishOnly": "npm run test",
+    "sized": "browserify src/internal/index.js -p [tsify --noImplicitAny] | wc -c",
```

Comment:
What's the purpose of this?

Reply:
i'll remove it. initially i was planning to use an `internal` directory instead of an npm package